src/Adapter/Attribute/CommandHandler/AddAttributeHandler.php

```diff
@@ -68,6 +70,13 @@ public function handle(AddAttributeCommand $command): AttributeId
 
         $id = $this->attributeRepository->add($attribute);
 
+        if (file_exists($command->getTextureFilePath())) {
```

Comment:
It should be the uploader that checks if the file exists, the handler is only here to check if the data exists (if it's not null) Besides I'm not sure but maybe `file_exists` doesn't like null input?